app/views/dashboard/table.scala.html

```diff
@@ -28,7 +29,7 @@
 
 @this(govukTable: GovukTable, link: link, pagination: pagination, paginationConfig: PaginationConfig)
 
-@(statusGroup: StatusGroup, pageOfSubmissions: PageOfSubmissions, baseHref: String)(implicit request: Request[_], messages: Messages)
+@(statusGroup: StatusGroup, pageOfSubmissions: PageOfSubmissions, baseHref: String)(implicit request: Request[_], messages: Messages, amendmentsConfig: DeclarationAmendmentsConfig)
```

Comment:
Couldn't `DeclarationAmendmentsConfig` directly be injected in `table.scala.html`? If not please ignore my comment, if yes, do you actually need it in `DashboardController`, `DashboardHelper` and `dashboard.scala.html`?

Reply:
Yeah that should work fine